README.md

```diff
@@ -99,6 +108,22 @@ pip3 install numpy
 After that is done downloading, move into the directory.  
 `cd automl-video-ondevice`
 
-## Running an Example
+## Run a TensorFlow-TensorRT Example
 
 `python3 examples/video_file_demo.py --model=data/traffic_model_trt.pb`
+
+## Run a TensorRT Example
+For TensorRT, the chip Engines converted with must be the same kind of chip to run inference workflow on. We hereby convert a TensorRT Engine on the Nano Platform. Later on, we hope to export the onnx model from Cloud console to hide the onnx converting steps from users.
+
+First convert the model to ONNX format, use the model downloaded from GCP Video Intelligence console:
+`python3 -m tf2onnx.convert --graphdef frozen_inference_graph.pb --inputs image_tensor:0 --outputs detection_boxes:0,detection_classes:0,detection_scores:0,num_detections:0 --output models/model_opset_10.onnx --fold_const --opset 10 --verbose`
+
+then, 
+`python3 examples/modify_onnx_model_opset10.py --onnx model_opset_10.onnx --modified model_opset_10_modified.onnx`
+
+Then convert ONNX graph to a TensorRT Engine file, please note the TensorRT conversion is a memory intensive process so be mindful when running on less powerful device like Nano:
+`/usr/src/tensorrt/bin/trtexec --onnx=model_opset_10_modified.onnx --fp16 --workspace=2048 --allowGPUFallback --saveEngine=vot_fp16.plan --verbose &> trtexec_build.log`
+
+The log could be viewed by `tail -f trtexec_build.log`. The TensorRT Engine file `vot_fp16.engine` could be deployed to a Deep Stream application.
```

Comment:
should this be vot_fp16.plan, instead of .engine from the command above?

Reply:
good catch, fixed...